pages/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/elasticsearch/index.md

```diff
@@ -0,0 +1,185 @@
+
+Full list of [Configuration Parameters](https://github.com/elastic/helm-charts/tree/master/elasticsearch#configuration) that can be applied to any of the current node roles:
+- [elasticsearch-client](https://github.com/mesosphere/kubeaddons-elastic/blob/elastic-7.10.x/values.yaml#L1)
+- [elasticsearch-master](https://github.com/mesosphere/kubeaddons-elastic/blob/dee7f94458532c6cd8802647f59e59f69d18f58f/values.yaml#L25)
+- [elasticsearch-data](https://github.com/mesosphere/kubeaddons-elastic/blob/dee7f94458532c6cd8802647f59e59f69d18f58f/values.yaml#L47)
+- [elasticsearch-ingest](https://github.com/mesosphere/kubeaddons-elastic/blob/dee7f94458532c6cd8802647f59e59f69d18f58f/values.yaml#L70)
+
+The current set of default parameters applied are found [here](https://github.com/mesosphere/kubeaddons-elastic/blob/elastic-7.10.x/values.yaml).
+### Updating Parameters
+
+Parameters can be updated by directly modifying them via the Kommander Catalog UI:
+
+**Example**: Increasing Data Node Replicas from the default to 4.
+
+1. Select `Edit Service` for the selected Elasticsearch instance.
+1. Modify `elasticsearch-data` to include the updated `replicas` count, as seen below:
+    ```
+    elasticsearch-data:
+    # It is suggested that this match the catalog addon name.
+    clusterName: "elasticsearch-oss"
+    # This must equal <elasticsearch-master.clusterName>-master
+    masterService: "elasticsearch-oss-master"
+    replicas: 4
+    ```
+1. Select `Save`
+
+    The output below shows that we now have 4 data nodes.
+    ```
+    $ kubectl get pods
+    NAME                         READY   STATUS    RESTARTS   AGE
+    elasticsearch-oss-client-0   1/1     Running   0          70m
+    elasticsearch-oss-data-0     1/1     Running   0          70m
+    elasticsearch-oss-data-1     1/1     Running   0          70m
+    elasticsearch-oss-data-2     1/1     Running   0          70m
+    elasticsearch-oss-data-3     1/1     Running   0          2m8s
+    elasticsearch-oss-master-0   1/1     Running   0          70m
+    elasticsearch-oss-master-1   1/1     Running   0          70m
+    elasticsearch-oss-master-2   1/1     Running   0          70m
+    ```
+### Monitoring
+
+Kommander includes Prometheus and Grafana as part of the federated [Workspace Platform Services](/dkp/kommander/1.3/workspaces/workspace-platform-services) along with [Centralized Monitoring](/dkp/kommander/1.3/centralized-monitoring/).
+
+The Kommander Catalog includes [Elasticsearch-Exporter](/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/elasticsearch-exporter/) to be used in conjunction with Elasticsearch to export metrics to Prometheus and Grafana.
+
+### External Access
+
+The Kommander Catalog includes [Kibana](/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kibana/) which provides a graphical way to view data stored in Elasticsearch
```

Comment:
Is the heading `External Access` appropriate here? Because there is no external access shown via NodePort or LoadBalancer.